draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1496,10 +1496,10 @@ Outputs:
 
 # TLS Servers {#tls_servers}
 
-TLS servers MUST use at least one of the three mechanisms listed below to
-present one or more SCTs from one or more logs to each TLS client during full
-TLS handshakes, where each SCT corresponds to the server certificate. TLS
-servers SHOULD also present corresponding inclusion proofs and STHs.
+CT-using TLS servers MUST use at least one of the three mechanisms listed below
```

Comment:
Nit: We've used the term CT-aware elsewhere in the document.

Reply:
I prefer "CT-using", which was EKR's proposal.  I'll use that throughout.